package.json

```
@@ -14,12 +15,11 @@
     },
     "description": "",
     "scripts": {
-        "build": "lerna run build",
+        "build": "yarn workspaces foreach run build",
         "lint": "eslint .",
-        "test": "lerna run test",
-        "postinstall": "npx husky install && npm run build",
-        "prepare": "node ./scripts/prepare.js",
-        "publish-workspaces": "npm publish --workspaces"
+        "test": "yarn workspaces foreach run test",
+        "postinstall": "npx husky install && yarn run build",
```

```suggestion
        "postinstall": "husky install && yarn run build",
```
https://typicode.github.io/husky/#/?id=install-1